### ISSUE TITLE
Question about flow field and grid sample

### ISSUE THREAD

**Issue by ShenZheng2000** [franciszzj/Leffa#28]
Thanks for your outstanding work! After reading your paper, I have a few questions. 

1. Is a flow field essentially a form of spatial cross-attention?

2. If the person is wearing a T-shirt and shorts and the garment is a dress, how does the flow field handle the alignment in such a scenario?

3. Does higher attention correspond to areas where the garment needs to warp more aggressively to align with the body pose?

**Comment by franciszzj** [comment]
Thank you for your attention to our work.

1. The flow field serves as an intermediate medium within the attention layer, allowing supervision without requiring additional labels, which makes it essential in this context.
2. During training, we use paired data, so this issue does not arise. During testing, the flow field is no longer needed, and only the standard attention operation is performed.
3. Could you elaborate further on this issue? I didn't quite grasp the specific point of your question.

**Comment by franciszzj** [comment]
Yes, this allows the model to extract more accurate features from the corresponding regions, better preserving detailed information at those regions.
